Packs/CommonScripts/Scripts/IsInternalDomainName/IsInternalDomainName.yml

```diff
@@ -0,0 +1,43 @@
+args:
+- auto: PREDEFINED
+  default: true
+  description: A comma-separated list of main - domains.
```

Comment:
@ShirleyDenkberg we want to emphasize that these are the domains we want to check (for example apps.google.com)